### ISSUE TITLE
NPE when removing virtual children in embedded apps

### ISSUE THREAD

**Issue by pleku** [vaadin/flow#7548]
When using a micro frontend setup with Flow, like the labs micro frontend demo, and a micro frontend is added & removed to the page due to user navigation, a later heartbeat to the removed micro frontend will cause a NPE:

```
Caused by: java.lang.NullPointerException
	at com.vaadin.flow.dom.ElementUtil.removeVirtualChildFromParent(ElementUtil.java:274)
	at com.vaadin.flow.component.webcomponent.WebComponentWrapper.lambda$disconnected$2af62eb9$1(WebComponentWrapper.java:123)
	at com.vaadin.flow.component.internal.UIInternals.lambda$setLastHeartbeatTimestamp$0(UIInternals.java:338)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at java.util.Collections$UnmodifiableCollection.forEach(Collections.java:1082)
	at com.vaadin.flow.component.internal.UIInternals.setLastHeartbeatTimestamp(UIInternals.java:338)
	at com.vaadin.flow.server.communication.HeartbeatHandler.synchronizedHandleRequest(HeartbeatHandler.java:67)
	at com.vaadin.flow.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)
	at com.vaadin.flow.server.VaadinService.handleRequest(VaadinService.java:1540)
```

This is related to the fix made in #7054

**Comment by pleku** [comment]
Fixed by #7061 which didn't have any ticket originally.

**Comment by pleku** [comment]
The fix has been included to Flow 2.1.3 which is in 14.1.4. It is also in prereleases for 14.2 & 15 since early January.
